e2e_test/bin/e2e_test.dart

```diff
@@ -0,0 +1,59 @@
+import 'dart:io';
+
+import 'package:http/http.dart';
+import 'package:sentry/sentry.dart';
+
+const _exampleDsn =
+    'https://8b83cb94764f4701bee40028c2f29e72@o447951.ingest.sentry.io/5428562';
+
+const _org = 'sentry-sdks';
+const _projectSlug = 'sentry-flutter';
+
+const _token = String.fromEnvironment('SENTRY_AUTH_TOKEN');
+
+void main(List<String> arguments) async {
+  print('Starting');
+  if (_token.trim().isEmpty) {
+    print('AUTH TOKEN is not set');
+    exit(1);
+  }
+  await Sentry.init((options) {
+    options.dsn = _exampleDsn;
+  });
+
+  final id = await Sentry.captureMessage('E2E Test Message');
```

Comment:
instead of a message, lets do an exception, which runs different parts of our code, well, ideally we do both, but lets start with `captureException`